### DISCUSSION TITLE
Why when I want to activate two-factor protection by Phone number I do not find my country Morocco (+212) ?

### DISCUSSION THREAD

**Discussion by anasse8** [community/community#147091] [Other Feature Feedback, Questions, & Ideas]
Why when I want to activate two-factor protection by Phone number I do not find my country Morocco (+212) ?

**Comment by mecodeatlas** [comment]
Hey @anasse8!

Thanks so much for reaching out in our community. If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

Let me know if you have any additional questions on this topic!

**Comment by mecodeatlas** [comment]
Hey, @anasse8 

You are more likely to get a useful response if you are posting your question in the applicable category, the Profile category is solely related to conversations around the **GitHub product Profile**. This question should be in the General category. I've gone ahead and moved it for you. Good luck!
